### PR TITLE
Tweaks and Fixes + Twinkle Doll Redo

### PR DESCRIPTION
**Pull request by MrVauxs**

### CODE REVIEW

File: collection/Jelke Ludolphij; Tavern of Trinkets.json [hunk 1]
```diff
@@ -422,16 +422,57 @@
 			}
 		},
 		{
-			"name": "Twinkle Doll",
+			"name": "Twinkle Doll ",
```
Comment (TheGiddyLimit) [inline collection/Jelke Ludolphij; Tavern of Trinkets.json]: Mind removing this trailing space?

Comment (TheGiddyLimit) [reply collection/Jelke Ludolphij; Tavern of Trinkets.json]: Eh whatever I'll do it later

Comment (MrVauxs) [reply collection/Jelke Ludolphij; Tavern of Trinkets.json]: It was to fix the {@i Twinkle Doll} issue, which is still not fixed.

Comment (TheGiddyLimit) [reply collection/Jelke Ludolphij; Tavern of Trinkets.json]: Yeah I saw that convo afterwards. True, it's not fixed _in production_ :wink:

**Pull request merged**